src/terminal/adapter/ITermDispatch.hpp

```diff
@@ -141,6 +141,8 @@ class Microsoft::Console::VirtualTerminal::ITermDispatch
                                        const DispatchTypes::DrcsFontUsage fontUsage,
                                        const VTParameter cellHeight,
                                        const DispatchTypes::DrcsCharsetSize charsetSize) = 0; // DECDLD
+
+    virtual StringHandler RequestSetting() = 0; // DECRQSS

```

Comment:
/cc @oising we were chatting about how to indicate what "sub-things" we support for things that have multiple .. well, sub-things.